docs/api-reference/overview.mdx

```diff
@@ -32,9 +32,16 @@ Example with the mem0 Python package:
 ```python
 from mem0 import MemoryClient
 
+# Recommended: Using organization and project IDs
 client = MemoryClient(
-  organization='YOUR_ORG_NAME',
-  project='YOUR_PROJECT_NAME',
+    org_id='YOUR_ORG_ID',
+    project_id='YOUR_PROJECT_ID',
+)
+
+# Legacy approach (deprecated, will be removed in v0.1.40)
+client = MemoryClient(
+    organization='YOUR_ORG_NAME',  # Deprecated
+    project='YOUR_PROJECT_NAME',   # Deprecated
```

Comment:
Don't mention this in docs. You probably want to encourage using the org_id and project_id